.github/workflows/release-step-3.yml

```diff
@@ -0,0 +1,69 @@
+name: "Release - Step 3: Release to CDN and Synchronize Branches"
+
+on:
+    workflow_dispatch:
+        inputs:
+            versionNumber:
+                description: 'Which SDK version to syncronize? [X.XX.XX]'
+                required: true
+                type: string
+
+jobs:
+    verify-version-number:
+      name:  Verify SDK Version (${{ github.event.inputs.versionNumber}})
+      runs-on: ubuntu-latest
+      steps:
+        # As a safe guard, if a valid release branch does not exist,
+        # this step will fail and throw an error.
+        # Pushing previous versions of the SDK will require manual intervention.
+        # This action is not used for rollbacks.
+        - name: Checkout SDK Version (${{ github.event.inputs.versionNumber}})
+          uses: actions/checkout@v3
+          with:
+            ref: release/${{ github.event.inputs.versionNumber}}
+
+
+    synchronize-branches:
+      name: Synchronize SDK to all branches
+      runs-on: ubuntu-latest
+      needs:
+        - verify-version-number
+      env:
+        GITHUB_TOKEN: ${{ secrets.MP_SEMANTIC_RELEASE_BOT }}
+        GIT_AUTHOR_NAME: mparticle-automation
+        GIT_AUTHOR_EMAIL: developers@mparticle.com
+        GIT_COMMITTER_NAME: mparticle-automation
+        GIT_COMMITTER_EMAIL: developers@mparticle.com
+
+      steps:
+        - name: Checkout master branch
+          uses: actions/checkout@v3
+          with:
+            fetch-depth: 0
+            repository: ${{ github.repository }}
+            token: ${{ secrets.MP_SEMANTIC_RELEASE_BOT }}
+            ref: master
+
+        - name: Merge release branch into master branch
+          run: |
+            git pull origin release/${{ github.event.inputs.versionNumber}}
+        
+        # We will eventually replace master with main
+        - name: Push release commits to main and master
+          run: |
+                git push origin HEAD:master
+                git push origin HEAD:main
```

Comment:
this is going to force the issue and push a main branch and we'll have both. can we wait to do this intentionally? try not to combine multiple things into one branch/pr like this.

Reply:
As per our slack convo, this is something we already do to keep things in sync until we can formally cut over to use `main` instead of `master`. I think we should force the switch over within the next month.